stan/math/rev/core/var.hpp

```diff
@@ -297,7 +200,7 @@ class var {
    * <i>Warning</i>: The returned result does not track changes to
    * this variable.
    */
-  inline vari* operator->() { return vi_; }
+  inline vari_type* operator->() { return vi_; }
```

Comment:
Weird operator

Reply:
Yeah, though idk if we use it anywhere or not